download.html

```diff
@@ -9,6 +9,8 @@ <h2>Download</h2>
       <div class="download-subheading">
         Please note that grin and grin-wallet come without a graphical user interface (GUI).<br>
         For GUI wallet and node see the community projects below.
+        </br></br>
+        <a href="docs.grin.mw/docs/getting-started/quickstart/install/">Quickstart↗</a>
```

Comment:
May I suggest something a bit different here?
```diff
❯ git diff _sass/download.scss 
diff --git a/_sass/download.scss b/_sass/download.scss
index 83d5a45..dbaf0ab 100644
--- a/_sass/download.scss
+++ b/_sass/download.scss
@@ -1,6 +1,6 @@
 .download-heading {
   text-align: center;
-  padding: 40px 20px 30px 20px;
+  padding: 40px 20px 0px 20px;
   background-color: white;
   p {
     max-width: 600px;
@@ -15,6 +15,10 @@
   text-align: center;
 }
 
+.download-quickstart {
+  padding: 10px;
+  text-align: center;
+}
 
 .download-section {
   max-width: 700px;
```

and

```diff
❯ git diff download.html 
diff --git a/download.html b/download.html
index e16581f..c21305d 100644
--- a/download.html
+++ b/download.html
@@ -9,10 +9,11 @@ layout: default
       <div class="download-subheading">
         Please note that grin and grin-wallet come without a graphical user interface (GUI).<br>
         For GUI wallet and node see the community projects below.
-        </br></br>
-        <a href="docs.grin.mw/docs/getting-started/quickstart/install/">Quickstart↗</a>
       </div>
     </div>
+    <div class="download-quickstart">
+      <a href="docs.grin.mw/docs/getting-started/quickstart/install/">Quickstart↗</a>
+    </div>
     <section class="download-section">
       <!-- Tab links -->
       <div class="tab-container">
```

That'll be a bit cleaner. I can commit directly to your branch if you agree.